pkg/codegen/python/gen.go

```diff
@@ -510,16 +552,26 @@ func (mod *modContext) genTypes(dir string, fs fs) error {
 		imports, inputSeen, outputSeen := imports{}, codegen.Set{}, codegen.Set{}
 		for _, t := range mod.types {
 			if input && mod.details(t).inputType {
-				visitObjectTypesFromProperties(t.Properties, inputSeen, func(t *schema.ObjectType) {
-					imports.addIf(mod, t.Token, true /*input*/, func(imp string) bool {
-						// No need to import `._inputs` inside _inputs.py.
-						return imp != "from ._inputs import *"
-					})
+				visitObjectTypesFromProperties(t.Properties, inputSeen, func(t interface{}) {
+					switch T := t.(type) {
+					case *schema.ObjectType:
+						imports.addIf(mod, T.Token, true /*input*/, func(imp string) bool {
```

Comment:
Should we rename `add` and `addIf` to `addType` and `addTypeIf`, now that we have `addResource`?